app/views/pages/privacy_policy.html.erb

```diff
@@ -0,0 +1,208 @@
+                        "https://www.zendesk.co.uk/company/privacy-and-data-protection/#faq-general-1" %>
+    </p>
+
+    <h3 class="govuk-heading-s">Website hosting services</h3>
+
+    <p class="govuk-body">We use GOV.UK PaaS to enable us to run the service.</p>
+
+    <p class="govuk-body">
+      <%= govuk_link_to "Visit the GOV.UK PaaS website to find out how they use and look after your data (“end user data”)",
+                        "https://www.cloud.service.gov.uk/privacy-notice/#what-data-we-collect-from-end-users" %>
+    </p>
+
+    <h3 class="govuk-heading-s">Error and activity logging services</h3>
+
+    <p class="govuk-body">
+      We need to keep some records so that we can identify and fix problems with the service. We use Sentry to
+      monitor errors and Logit to keep activity logs.
+    </p>
+
+    <p class="govuk-body">
+      We do not identify you from these records unless needed - for example, to investigate an issue you raise.
+    </p>
+
+    <p class="govuk-body">
+      <%= govuk_link_to "Visit the Sentry website to read about their security practices", "https://sentry.io/security/" %>
+    </p>
+
+    <p class="govuk-body">
+      <%= govuk_link_to "Visit the Logit website to read about their security practices", "https://logit.io/security" %>
+    </p>
+
+    <h3 class="govuk-heading-s">User research tools</h3>
+
+    <p class="govuk-body">
+      If you agree to take part in user research, we’ll ask if you consent to use Lookback to carry out and
+      record a video session with you.</p>
+
+    <p class="govuk-body govuk-!-margin-bottom-7">
+      <%= govuk_link_to "Visit Lookback’s website to read about how they use your data (as a “participant”)", "https://lookback.io/privacy/" %>
+    </p>
+
+    <h2 class="govuk-heading-m" id="how-long-we-keep-your-personal-data">How long we keep your personal data</h2>
+
+    <p class="govuk-body">We keep your personal data for as long as you use the service.</p>
+
+    <p class="govuk-body">
+      We may also keep it for some time after you stop using the service. This is so that we can analyse any
+      feedback you provided to improve the service.
+    </p>
+
+    <p class="govuk-body govuk-!-margin-bottom-7">
+      We’ll keep your data for no longer than 7 years after you stop using the service.
+    </p>
+
+    <h2 class="govuk-heading-m" id="raise-a-concern">Raise a concern</h2>
+
+    <p class="govuk-body">
+      You have rights under the Data Protection Act 2018, such as the right to ask for your data to be erased.
+    </p>
+
+    <p class="govuk-body">
+      <%= govuk_link_to "Find out about your rights by visiting the Information Commissioner’s Office (ICO) website", "https://ico.org.uk/" %>
+    </p>
+
+    <p class="govuk-body">
+      Contact the <%= t('service_name') %> team at
+      <%= govuk_mail_to "registerateacher@digital.education.gov.uk" , "registerateacher@digital.education.gov.uk", subject: "My personal data" %>
```

Comment:
@EmmaFrith thanks for pointing out the long encrypted test you were seeing on the page. I've fixed this now but it was basically presetting the subject of the email if the user clicked on the link.